### DISCUSSION TITLE
Unity not opening after recent commit c7f4a7c

### DISCUSSION THREAD

**Discussion by rajas1310** [empriselab/RCareWorld#113] [Help - Setting up / Configuring] [answered]
Hey,
I was able to run the scripts successfully using `python test_bathing_scene.py --graphics`
But, after pulling the recent commit (c7f4a7c), the Unity window won't open and with the same command.
The script is stuck on `Waiting for connections on port: 5005...`

**Comment by shxingch** [comment, marked as answer]
Hi @rajas1310 ! We have updated the dressing scene and you can try it again!

**Reply by rajas1310** [reply]
Thank you!
